### ISSUE TITLE
mail collector: content inside blockquote is ignored

### ISSUE THREAD

**Issue by asabatep** [glpi-project/glpi#17577]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.16

### Bug description

HTML mail content inside ```<blockquote>``` tags appear to be ignored by the mail collector, and do not appear in the resulting ticket.

This is unfortunate since after an update our mail client wraps all forwarded content inside blockquote tags, and so empty tickets are created.

### Steps To reproduce

1. Send email including content within ```<blockquote>``` tags.
2. Ticket is created with the content in ```<blockquote>``` tags ignored.

![not-working](https://github.com/user-attachments/assets/99eed920-6aaf-4054-aeaa-128e97fb5188)
![working](https://github.com/user-attachments/assets/22c00c43-7ed4-487d-b3f3-eb42a65acce2)
[not-working.eml.txt](https://github.com/user-attachments/files/16334368/not-working.eml.txt)
[working.eml.txt](https://github.com/user-attachments/files/16334369/working.eml.txt)

### Your GLPI setup information

Operating system: Linux glpi.xxxx.local 4.18.0-513.24.1.el8_9.x86_64 #​1 SMP Thu Apr 4 18:13:02 UTC 2024 x86_64
PHP 8.2.19 fpm-fcgi (Core, PDO, Phar, Reflection, SPL, SimpleXML, Zend OPcache, apcu, bz2, calendar, cgi-fcgi, ctype, curl,
	date, dom, exif, fileinfo, filter, ftp, gd, gettext, hash, iconv, igbinary, intl, json, ldap, libxml, mbstring, msgpack, mysql,
	mysqli, mysqlnd, openssl, pcre, pdo_mysql, pdo_sqlite, random, redis, session, soap, sockets, sodium, sqlite3, standard,
	tokenizer, xml, xmlreader, xmlrpc, xmlwriter, xsl, zip, zlib)
Setup: max_execution_time="30" memory_limit="128M" post_max_size="50M" safe_mode="" session.save_handler="files"
	upload_max_filesize="50M" disable_functions="" 
Software: nginx/1.14.1
	Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/114.0
Server Software: MariaDB Server
	Server Version: 11.4.2-MariaDB-log
	Server SQL Mode: STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION
	Parameters: glpi@127.0.0.1/glpi
	Host info: 127.0.0.1 via TCP/IP
	
PHP version (8.2.19) is supported.PHP version (8.2.19) is supported.
Sessions configuration is OK.Sessions configuration is OK.
Allocated memory is sufficient.Allocated memory is sufficient.
mysqli extension is installed.mysqli extension is installed.
Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.Following extensions are installed: dom, fileinfo, filter, libxml, json, simplexml, xmlreader, xmlwriter.
curl extension is installed.curl extension is installed.
gd extension is installed.gd extension is installed.
intl extension is installed.intl extension is installed.
zlib extension is installed.zlib extension is installed.
The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.The constant SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_NPUBBYTES is present.
Database engine version (11.4.2) is supported.Database engine version (11.4.2) is supported.
No files from previous GLPI version detected.No files from previous GLPI version detected.
The log file has been created successfully.The log file has been created successfully.
Write access to /var/www/html/files/_cache has been validated. Write access to /var/www/html/files/_cron has been validated. Write access to /var/www/html/files has been validated. Write access to /var/www/html/files/_dumps has been validated. Write access to /var/www/html/files/_graphs has been validated. Write access to /var/www/html/files/_lock has been validated. Write access to /var/www/html/files/_pictures has been validated. Write access to /var/www/html/files/_plugins has been validated. Write access to /var/www/html/files/_rss has been validated. Write access to /var/www/html/files/_sessions has been validated. Write access to /var/www/html/files/_tmp has been validated. Write access to /var/www/html/files/_uploads has been validated.Write access to /var/www/html/files/_cache has been validated.
Write access to /var/www/html/files/_cron has been validated.
Write access to /var/www/html/files has been validated.
Write access to /var/www/html/files/_dumps has been validated.
Write access to /var/www/html/files/_graphs has been validated.
Write access to /var/www/html/files/_lock has been validated.
Write access to /var/www/html/files/_pictures has been validated.
Write access to /var/www/html/files/_plugins has been validated.
Write access to /var/www/html/files/_rss has been validated.
Write access to /var/www/html/files/_sessions has been validated.
Write access to /var/www/html/files/_tmp has been validated.
Write access to /var/www/html/files/_uploads has been validated.
For security reasons, SELinux mode should be Enforcing.For security reasons, SELinux mode should be Enforcing.

Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.Web server root directory configuration is not safe as it permits access to non-public files. See installation documentation for more details.
GLPI data directories are located in a secured path.GLPI data directories are located in a secured path.
Sessions configuration is secured.Sessions configuration is secured.
OS and PHP are relying on 64 bits integers.OS and PHP are relying on 64 bits integers.
exif extension is installed.exif extension is installed.
ldap extension is installed.ldap extension is installed.
openssl extension is installed.openssl extension is installed.
Following extensions are installed: bz2, Phar, zip.Following extensions are installed: bz2, Phar, zip.
Zend OPcache extension is installed.Zend OPcache extension is installed.
Following extensions are installed: ctype, iconv, mbstring, sodium.Following extensions are installed: ctype, iconv, mbstring, sodium.
Write access to /var/www/html/marketplace has been validated.Write access to /var/www/html/marketplace has been validated.
Timezones seems loaded in database.Timezones seems loaded in database.

**Comment by cconard96** [comment]
I could not even get the provided non-working email to generate a ticket. I added it to the tests we have for the mail collector and it is refusing the import for the email.
It seems the Date header was removed and the From address was made invalid when the content was redacted to remove sensitive details. Assuming they were present before, I added/fixed those issues manually.

After that, the test passed and confirmed that a ticket was created with the `<blockquote>` element and its content. I then manually tested creating a ticket and copying the email content into the "Source Code" for the ticket description editor and it added the blockquote properly.

I cannot seem to recreate the issue.
